### PR TITLE
Add QuTiP to compare QHSO and the transmon

### PR DESCRIPTION
**Pull request by DRA-chaos**
**Description**
Small phrasing improvements to the README.

Was: ~~Added a utility comparison between the QHSO and Transmon~~

**Changelog**
Small phrasing improvements to the README.

### CODE REVIEW

**Comment by Ericgig** [comment]
Hi DRA-chaos, thank you for the PR, but we keep notebooks example in another repo: https://github.com/qutip/qutip-notebooks. Could you move your PR there. Also please don't use space in file names.

The wording improvement in the readme is nice, can you leave just those changes here.

File: README.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ and [J. R. Johansson](https://github.com/jrjohansson)
 [![PyPi Downloads](https://img.shields.io/pypi/dm/qutip?label=downloads%20%7C%20pip&logo=PyPI)](https://pypi.org/project/qutip)
 [![Conda-Forge Downloads](https://img.shields.io/conda/dn/conda-forge/qutip?label=downloads%20%7C%20conda&logo=Conda-Forge)](https://anaconda.org/conda-forge/qutip)
 
-QuTiP is open-source software for simulating the dynamics of closed and open quantum systems.
-The QuTiP library uses the excellent Numpy, Scipy, and Cython packages as numerical backend, and graphical output is provided by Matplotlib.
+QuTiP is an open-source software for simulating the dynamics of closed and open quantum systems.
+The QuTiP library uses Numpy, Scipy, and Cython packages as numerical backends, and Matplotlib for graphical output.
```
Comment (hodgestar) [inline README.md:25]:
```suggestion
QuTiP is open-source software for simulating the dynamics of closed and open quantum systems.
It uses the excellent Numpy, Scipy, and Cython packages as numerical backends, and graphical output is provided by Matplotlib.
```
"software" is an uncountable noun and so does not take an article. Praising numpy, SciPy and Cython shows we're grateful to those projects and sets a good tone, so I'd like to keep it. Using a slightly different phrasing when mentioning Matplotlib makes the sentence less repetitive, so I'd like to keep it that way.

File: README.md [hunk 2]
```diff
@@ -21,11 +21,11 @@ and [J. R. Johansson](https://github.com/jrjohansson)
...
 QuTiP aims to provide user-friendly and efficient numerical simulations of a wide variety of quantum mechanical problems, including those with Hamiltonians and/or collapse operators with arbitrary time-dependence, commonly found in a wide range of physics applications.
 QuTiP is freely available for use and/or modification, and it can be used on all Unix-based platforms and on Windows.
-Being free of any licensing fees, QuTiP is ideal for exploring quantum mechanics in research as well as in the classroom.
+QuTiP does not charge any licensing fees and hence is ideal for exploring quantum mechanics in research as well as in the classroom.
```
Comment (hodgestar) [inline README.md:28]:
```suggestion
Being free of any licensing fees, QuTiP is ideal for exploring quantum mechanics in research as well as in the classroom.
```
I think the new phrasing is a slight degradation, so I'd like to keep the original.

**Review by hodgestar — CHANGES_REQUESTED** [review]
Thank you for removing the notebook -- is there a qutip-notebook PR for it yet?

I declined most of the changes left, but not all of them.

**Review by hodgestar — APPROVED** [review]

**Comment by hodgestar** [comment]
@DRA-chaos Thank you for the small README fixes. Looking forward to the qutip-notebook PR.

**Pull request merged**